func/python/env_check.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
```

Comment:
This test was missing before, just checking the Python interaction with the environment variables (again, something I thought was a problem working through the bug)